app/assets/stylesheets/components.scss

```diff
@@ -16,26 +16,12 @@
 }
 
 // Home ==================================================================
-.dgu-home-search {
-  padding-top: 2.5em;
-}
-
 .dgu-topics {
-
-  margin-top: 2em;
-
-  h2 {
-    font-weight: bold;
-  }
-
-  @media (min-width: 641px) {
-    li {
-      min-height: 6em;
-    }
+  &__heading {
```

Comment:
If this is only going to be one rule could it just be written as `.dgu-topics__heading`?